great-idea-website/css/index.css

```diff
@@ -63,4 +63,116 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+	width: 876px;
+	margin: auto;
```

Comment:
You will learn more about this next week, but I would do a `margin: 0 auto;` here. It will make it so the height is at a constant 0 and the width is dynamic. You will know more next week!